res/ai/DefaultAIPlayer/TaskNewsAgency.lua

```diff
@@ -441,12 +449,20 @@ function JobNewsAgency:GetNewsList(paidBonus)
 		table.insert(currentNewsList, news)
 	end
 
+	local cultureAward = 0
+	if getPlayer().currentAwardType == TVT.Constants.AwardType.CULTURE then
+		cultureAward = 1
+	end 
 
 	-- sort by attractivity modifed by paid-state-bonus
 	-- precache complex weight calculation
 	local weights = {}
 	for k,v in pairs(currentNewsList) do
-		weights[ v.GetID() ] = v.GetAttractiveness() * (1.0 + v.IsPaid() * paidBonus)
+		local weight = v.GetAttractiveness() * (1.0 + v.IsPaid() * paidBonus)
+		if cultureAward == 1 and v.GetGenre() == TVT.Constants.NewsGenre.CULTURE then
+			weight = weight * 3
```

Comment:
Das koennte man theoretisch in eine "individuelle" Variable ueberfuehren - also "charakterabhaengig" bzw "erfahrungsbasiert" anpassen.

Reply:
Ja, das Rausziehen von Konstanten ist ja ohnehin auf der Aufräumliste für die KI.
Tatsächlich bin ich mir aber gar nicht so sicher, wie sinnvoll charakterabhängige Werte sind. Der "normale" Zufall (welche Lizenzen bekomme ich, welche Werbung wird mir angeboten) hat schon so starken Einfluss, dass sich *kleine* Parameteranpassungen kaum auswirken. Auch ist es fraglich, ob dauerhaft gültige Charakterwerte sinnvoll sind.